app/Commands/Dev/Geb/Infer.hs

```diff
@@ -27,7 +27,6 @@ runCommand opts = do
             Geb.ppOut
               opts
               (tyMorph ^. Geb.typedMorphismObject)
-          embed $ putStrLn ""
```

Comment:
Why removing newline here (not only embed)?

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582